prerender/storePrerenderedContent.js

```diff
@@ -86,6 +86,15 @@ function log(message, ...args) {
   console.log(`[storePrerenderedContent] ${message}`, ...args);
 }
 
+async function startBrowser() {
+  try {
+    return await puppeteer.launch();
+  } catch (e) {
+    reportError("Could not start browser!", e);
+    process.exit(1);
```

Comment:
IIRC, using `process.exit` may lead to loss of log output. So, this fix kinda defeats the purpose. We should use the same strategy as below, with `process.exitCode = 1;` and ending the script gracefully.

Reply:
Good point.